README.md

```diff
@@ -61,6 +61,34 @@ class TestCustomerModel(TestCase):
 
 Check out [documentation](<http://model-bakery.readthedocs.org/>) for more complete examples.
 
+## Contributing
+
+1. Fork this repo and clone it to your computer:
+
+```
+$ git clone git@github.com:YOUR_USER/model_bakery.git
+```
```

Comment:
Maybe also forking itself we can add as a step or link:
https://github.com/model-bakers/model_bakery/fork

Reply:
Fixed by 921155be8303d2aee140ccf13b157b2a426cb252